test/integration/job/job_test.go

```diff
@@ -125,17 +125,6 @@ func TestMetrics(t *testing.T) {
 	job_index := 0 // job index to avoid collisions between job names created by different test cases
 	for name, tc := range testCases {
 		t.Run(name, func(t *testing.T) {
-
-			// record the metrics after the job is created
-			jobFinishedNumBefore, err := getCounterMetricValueForLabels(metrics.JobFinishedNum, tc.wantJobFinishedNumMetricDelta.Labels)
-			if err != nil {
-				t.Fatalf("Failed to collect the JobFinishedNum metric before creating the job: %q", err)
-			}
-			jobPodsFinishedBefore, err := getCounterMetricValueForLabels(metrics.JobPodsFinished, tc.wantJobPodsFinishedMetricDelta.Labels)
-			if err != nil {
-				t.Fatalf("Failed to collect the JobPodsFinished metric before creating the job: %q", err)
-			}
-
 			// create a single job and wait for its completion
```

Comment:
Call the `resetMetrics` function

Reply:
oops, it was passing locally because the cases use different labels